data_summaries/Cidar.json

```diff
@@ -0,0 +1,32 @@
+{
+  "dataset-name (must match unique identifier)": {
+    "Unique Dataset Identifier": "cidar",
+    "Dataset Name": "cidar",
+    "Paper Title": "CIDAR: Culturally Relevant Instruction Dataset For Arabic",
+    "Dataset URL": "https://github.com/ARBML/CIDAR",
+    "GitHub URL": "https://github.com/ARBML/CIDAR",
+    "Hugging Face URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Papers with Code URL": "https://paperswithcode.com/dataset/cidar",
+    "ArXiv URL": "https://arxiv.org/abs/2402.03177",
+    "Semantic Scholar Corpus ID": "",
+    "Collection": "Cidar",
+    "Collection URL": "https://hf.co/datasets/arbml/CIDAR",
+    "Languages": ["Arabic"],
+    "Task Categories": ["Instruction Generation"],
+    "Text Sources": ["huggingface.co/datasets/mlabonne/alpagasus"],
+    "Model Generated": ["GPT-3.5 Turbo"],
+    "Format": ["Zero-shot"],
+    "Human Annotation": "Yes",
+    "Derived from Datasets": ["Alpagasus"],
+    "Creators": ["ArbML"],
+    "Licenses": [
+      {
```

Comment:
Let's add the OpenAI terms of use to this list (should be in the license classes file), given that the dataset was model generated.